AmethystAPI/src/minecraft/src-client/common/client/gui/gui/VisualTree.hpp

```diff
@@ -4,5 +4,5 @@
 class VisualTree {
 public:
     std::byte padding[8];
-    UIControl* mRootControlName; // this + 8
```

Comment:
I generally think we should avoid renaming vanilla variable names unless there is a reason to, because it could lead to wasting time if e.g. its not clear that we made an arbitrary choice to rename it, and therefore it no longer lines up with the game

Reply:
This was an accident. I'll rename it back